app/views/users/show.html.erb

```diff
@@ -24,7 +24,8 @@
 	<% if @user.id == session[:user_id] %>
 		<%= link_to "View Uploaded Documents", documents_path %>  | 
 		<%= link_to "Edit Your Profile", edit_user_path(@user) %>  | 
-		<%= link_to "Privacy Settings", edit_privacy_path(@user) %>   
+		<%= link_to "Privacy Settings", edit_privacy_path(@user) %> |
+		<%= link_to "Change Your Password", editpassword_path %>  
 	<% end %>
 	<% if !@ambassador && @user.id == current_user.id %>
 		<%= link_to "Become an Ambassador", new_role_path %>
```

Comment:
I'm going to add a " | " here to fix the two links running together 
![image](https://cloud.githubusercontent.com/assets/4001044/20440017/e3b18ea4-ad83-11e6-9c19-0c1944677c17.png)